bower.json

```diff
@@ -17,8 +17,8 @@
     "ez-js-rest-client": "ezsystems/ez-js-rest-client",
     "yui3": "http://yui.zenfs.com/releases/yui3/yui_3.18.1.zip",
     "flag-icon-css": "~0.6.4",
-    "alloy-editor": "ezsystems/alloy-editor#~0.6.0",
-    "widget": "http://download.ckeditor.com/widget/releases/widget_4.5.3.zip",
-    "lineutils": "http://download.ckeditor.com/lineutils/releases/lineutils_4.5.3.zip"
+    "alloy-editor": "liferay/alloy-editor#~0.7.5",
+    "widget": "http://download.ckeditor.com/widget/releases/widget_4.5.5.zip",
```

Comment:
Interesting question :) AlloyEditor 0.7.5 embeds CKEditor 4.5.5 that's why I choose 4.5.5 as the version of the plugins but maybe it would be better to pick the last for the plugins. I need to check.